go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/joho/godotenv v1.3.0 h1:Zjp+RcGpHhGlrMbJzXTrZZPrWj+1vfm90La1wgB6Bhc=
```

Comment:
@dm36, should you store `go.sum` too? Does it matter? If no need, just add this into `.gitignore`

Reply:
@smusali https://github.com/kubernetes/kubernetes tracks their .gosum in git

Reply:
as does https://github.com/elastic/beats